src/utils/jest-config.ts

```diff
@@ -37,9 +37,10 @@ export function modifyJestConfig(jestConfigPath: string, data: any) {
 
     fs.writeFileSync(jestConfigPath, JSON.stringify(adjustedJestConfig, null, 2));
   } catch (error) {
-    cli.warn(error);
-    return cli.warn(
-      `Could not edit your Jest config at "${jestConfigPath}". Please verify if the location is correct and consider opening a bug ticket at github.com/SAP/cloud-sdk-cli`
+    recordWarning(
+      `Could not edit your Jest config at "${jestConfigPath}".`,
+      'Please verify if the location is correct and consider opening a bug ticket',
+      'at https://github.com/SAP/cloud-sdk-cli'
```

Comment:
Maybe add a dot in the end of the sentence?

Reply:
Left out to make it easier to copy